scripts/output_scripts/slack.py

```diff
@@ -89,10 +89,16 @@ def main(args: Namespace) -> None:
             if details:
                 fail_status += f"\t•  {app_name}\n"
                 if isinstance(details, dict):
+                    device_status = None
```

Comment:
```suggestion
                    device_status = ""
```
So that the type is always `str`, and `device_status` is still `False` if it is evaluated empty in a `if` condition.